### PR TITLE
Convert menu in switch wallet to the new context menu

### PR DESCRIPTION
**Pull request by tchayen**
Fixes RNBW-4106
Figma link (if any):

## Screen recordings / screenshots

On iOS nothing changed:

https://user-images.githubusercontent.com/5769281/182343810-28f077f1-6f80-41d7-bfcb-64344b358afe.MP4

On Android the new context menu is used:

https://user-images.githubusercontent.com/5769281/182611443-59997bac-f597-48ff-9300-e5ba81ac37a9.mp4

## What to test

Test all possible cases of editing wallets:
- On iOS it's possible to edit wallet by long pressing even not in edit mode (on Android not).
- It's not allowed to remove wallet when there's only one left.
- And many more which I probably don't know 😬.

## Final checklist

- [x] Assigned individual reviewers?
- [x] Added labels? (`team1/team2`, `critical path`, `release`, `dev QA`)
- [x] Did you test both iOS and Android?
- [x] If your changes are visual, did you check both the light and dark themes?
- [x] Added e2e tests? If not, please specify why
- [x] If you added new critical path files, did you update the CODEOWNERS file?
- [ ] If no `dev QA` label, did you add the PR to the QA Queue?

### CODE REVIEW

**Comment by terrysahaidak** [comment]
on Android context menu should appear from the button that triggers it, on the video its placement is confusing.

**Comment by tchayen** [comment]
The problem here is that the whole row acts as a button (even though it looks like the button is only on the left).

**Comment by terrysahaidak** [comment]
@tchayen can you try that prop that tells where to render the menu?

**Comment by tchayen** [comment]
@terrysahaidak 

Screenshot_20220803-130401_Rainbow

**Review by terrysahaidak — APPROVED** [review]
Now it's better

**Comment by osdnk** [comment]
https://streamable.com/z5kj6b

This is very wrong. Let's stick the menu to the small button, not the whole row

**Review by osdnk — CHANGES_REQUESTED** [review]
^

**Review by osdnk — APPROVED** [review]

File: src/components/change-wallet/AddressRow.js [hunk 1]
```diff
@@ -151,76 +153,131 @@ export default function AddressRow({
...
+          enableHapticFeedback={!editMode}
+          onLongPress={!watchOnly ? showIOSMenu : onPress}
+          onPress={editMode ? showIOSMenu : onPress}
+          scaleTo={editMode ? 1 : 0.98}
+        >
+          {content}
+        </ButtonPressAnimation>
+      ) : (
+        content
+      )}
```
Comment (perunt) [inline src/components/change-wallet/AddressRow.js:280]: It looks like there is no way to choose another wallet on android

**Review by perunt — CHANGES_REQUESTED** [review]
the new context menu works fine
just one comment about the ability to change wallets

**Review by perunt — APPROVED** [review]
:rocket:

**Pull request merged**